grails-app/services/org/pih/warehouse/picklist/PicklistService.groovy

```diff
@@ -86,6 +87,34 @@ class PicklistService {
         productAvailabilityService.refreshProductsAvailability(orderItem?.order?.origin?.id, [orderItem?.product?.id], binLocations?.unique(), false)
     }
 
+    void clearPicklist(String id) {
```

Comment:
This is the second `void clearPicklist(String id)` in the code. The other one is in the `StockMovementService`. Please add a comment explaining that the parameter with id here is meant to be picklistItem id, and in the other one that it expects stock movement id. I would even consider deprecating the other one, but we have not discussed it so it is just a thought.

Reply:
I will add the comments, but imho it's kinda self-explonatory that if I use the `PicklistService`, we are rather referring to the picklist id (unless the argument was named differently), and the same with the `StockMovementService` - if I use `id` as an argument, it's rather the SM id.

Reply:
Technically it is self-explanatory, but I feel like there was a lot of self-explanatory code in this repo, that is now not that self-explanatory :sweat_smile:

Reply:
To be honest I expected something more like a javadoc with `[at]param`, but that can be adjusted in the next pr from that epic.

Reply:
yeah, you are right, I will try to remember to change it to the javadoc.